project.md

```diff
@@ -58,11 +179,32 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+I think you'd be able to engage with the language quite frequently because the
+whole point is to create some way of transcribing taiko from something that is
+only heard to something that can be written down. There are already conventions
+in place that will make this easier, as shown in the link further above, but
+adjustments must still be made to allow for a fluent transcription.
+
 ### Scope
 
 _How big an idea is this? How ambitious is this project?_
 
+Because taiko is percussive, it's a less complicated to deal with than other
+instruments. I think it is very doable because it's a simple concept that I can
+always add onto if I have more time such as including dynamics, standard sheet
+music notation like repeats, and if I were really ambitious, trying to figure
+out a way to include the dancing aspect of taiko (taiko is not a stationary art;
+it is largely performance through dancing and shouting).
```

Comment:
This dancing and shouting part of taiko sounds cool! If you wanted to implement this, maybe a library of stick figures would work?